README.md

```
@@ -12,6 +21,44 @@ The `CodeRev` is a web-based application designed to help programmers improve th
+5. Now run the application backend after returning to `/backend` directory
+
+```bash
+dotnet watch run
+```
+6. Open-up a new terminal and run the database migration 
+
+```bash
+dotnet ef migrations add <migration_name> -c <context_name> -o <absolute_path_of_the_folder>
+dotnet ef database update -c <context_name>
+```
+7. You can verify database table `Solves` by entering into the `docker-container`
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 7; Style: 1/1/1

(MD029, ol-prefix)